src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_BALL_COUNT = 100;
```

Comment:
```suggestion
    private static final int MAX_BALL_NUMBER = 100;
```

we can make this constant private as we use it only in this class